### ISSUE TITLE
freetype.dll error with win7-64

### ISSUE THREAD

**Issue by TomQv** [Robmaister/SharpFont#119]
I'm using the 64/freetype6.dll provided with this package, and it runs fine on Win10-64.
Same app installed on Win7-64 gives windows-error  on start:

"Dll "freetype6": Module not found. (Exception HRESULT 0x8007007E) can't load"

https://aka.ms/vs/15/release/vc_redist.x64.exe is installed on that machine.
Any ideas? 

Tom

**Comment by Robmaister** [comment]
Most likely a duplicate of #99 

Haven't had much time to be a maintainer, it's only an issue if you're using the .props file that comes with the Nuget package

**Comment by TomQv** [comment]
I'm not using the .props file that comes with the Nuget package. But I'm using the freetype6.dll thats included in the package. 
Using [ProcessMon ](https://docs.microsoft.com/en-us/sysinternals/downloads/procmon)I figured out, that freetype6.dll is depending on msvcrt120.dll. Just adding this dll to the output-folder solves the issue.

**Comment by Robmaister** [comment]
Makes sense, I'm probably going to publish versions of freetype with a statically linked CRT in the future to avoid the compatibility matrix entirely. Only just recently found out that it's not as big of an issue to use the statically linked CRT as a lot of people make it out to be.

**Comment by HinTak** [comment]
FWIW, the {Win32, Win64} directories in
https://github.com/HinTak/Font-Validator/tree/master/bin

contains mingw-built freestype (so only depends on msvcrt.dll, I think). Those are what I ship with font validator (which uses SharpFont) with some enhancements which should be entirely compatible . The additional enhancement does not change current behavior but strictly additions - supposedly slightly slower but I have been running my fully augmented freetype as system-wise on fedora for a month and has not noticed any significant slow down.
    On Saturday, 23 June 2018, 02:17:52 GMT+8, Robert Rouhani <notifications@github.com> wrote:  
 
 
Makes sense, I'm probably going to publish versions of freetype with a statically linked CRT in the future to avoid the compatibility matrix entirely. Only just recently found out that it's not as big of an issue to use the statically linked CRT as a lot of people make it out to be.
